eclair-core/src/main/resources/reference.conf

```diff
@@ -151,6 +151,8 @@ eclair {
       max-total-pending-channels-private-nodes = 99 // maximum number of pending channels we will accept from all private nodes
       channel-opener-whitelist = [] // a list of public keys; we will ignore rate limits on pending channels from these peers
     }
+
+    quiescence-timeout = 2 minutes // maximum time we will wait for quiescence to complete before disconnecting
```

Comment:
nit: 1 minute is probably more than enough here, we're just exchanging `commit_sig` / `revoke_and_ack` in both directions and nothing more.